activity_calendar/templates/activity_calendar/activity_overview.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'core/base.html' %}
+{% load static %}
+{% load martor_render %}
+
+{% block title %}
+    Squire - Boardgames
+{% endblock title %}
+
+{% block content-frame-class %}
+    container-fluid
+{% endblock %}
+
+{% block content %}
+    <div class="m-auto" style="max-width: 1250px;">
+        <h1>Activities</h1>
```

Comment:
I suggest renaming the title to "Upcoming Activities", and to add an "All Activities"-button as I feel like this link is a bit too hidden.